content/blog/tina-is-in-beta.md

```diff
@@ -0,0 +1,135 @@
+5. Create an admin route to allow people to edit, and a way to exit.
+6. Create a schema file ready for you to shape your content
+
+This allows you to move quickly and experience Tina without having to write any code. Then when you are ready you can easily extend it to use parts of your existing site.
+
+### Improving and Adding Guides
+
+When we introduced Tina, we had a single guide that got you up and running with our Tina Cloud Starter. This was a great way for users to experience Tina but we found that people were missing some key concepts of Tina.
+
+I went back to the drawing board and created a new guide that takes the [Next.js Starter Blog and adds Tina and Tina Cloud](/guides/tina-cloud/add-tinacms-to-existing-site/overview/) to it while explaining each concept as we went. This feels like a perfect way to show off Tina, learn how to use Tina, with something that many users are familar with.
+
+We also removed old guides that no longer promote Tina's best practices and moved some of our other guides into our experimental section. Experimental to us means that we can't guarantee that there won't be bugs or issues with the packages used. 
+
+## Improving our Tina Starter
+
+
+The Tina Starter was built originally to show "the power of Tina" while it did that, we didn't feel that it showed a real-world example. So we went back to the drawing board and created our new [Tina Starter](/guides/tina-cloud/starter/overview/), which includes a landing page, blog, and about pages. You can edit and rearrange the content and we styled it with TailwindCSS to give it some extra shine! Below is an example of just some of the work you can do:
+
+
+![Quickstart Example](/img/edit-alongside-content.gif)
+
+## Media Manager
+
+Media manager was one of the most important features that we needed for Tina Cloud. Our Cloudinary Media manager allows you to change images, upload new images, and delete ones you no longer need without ever leaving the Tina editing experience. 
+
+I wrote a [blog post announcing it](/blog/manage-your-media-with-cloudinary/) and how to implement it into your application.
+
+## Caching Improvements
+
+Speed and performance have been something we have been actively working on. We introduced some improvements behind the scenes to improve the way we retrieve the data for your site. Tina was already performance first but now it's even better! 
+
+## Creating `@tinacms/toolkit`
+
+TinaCMS was built with small modular packages, this meant that we relied heavily on React context. The dependency mismatches from over-modularizing our toolkit,  led to many bugs related to missing context.
+
+Our open-source team created @tinacms/toolkit which incorporates the essentials of Tina all in one place. This simplifies everything for you as a user and Tina as a product.
+
+You can read about all the updates and why decided to make the changes in our pinned [GitHub issue](https://github.com/tinacms/tinacms/issues/1898).
+
+## Vercel Integration
```

Comment:
Vercel integration is currently available when you add our starter, we're already working on providing a Vercel deploy button for any app added to Tina Cloud.